crates/task-impls/src/consensus.rs

```diff
@@ -360,13 +361,14 @@ where
 }
 
 impl<
-        TYPES: NodeType<BlockHeader = VIDBlockHeader, BlockPayload = VIDBlockPayload>,
+        TYPES: NodeType<BlockHeader = H, BlockPayload = VIDBlockPayload>,
         I: NodeImplementation<
             TYPES,
             Leaf = SequencingLeaf<TYPES>,
             ConsensusMessage = SequencingMessage<TYPES, I>,
         >,
         A: SequencingConsensusApi<TYPES, SequencingLeaf<TYPES>, I> + 'static,
+        H: BlockHeader<Payload = VIDBlockPayload>,
```

Comment:
This parameter seems unnecessary, I think we can delete `BlockHeader = H` above and instead add a `where TYPES::BlockHeader: BlockHeader<Payload = VIDBlockPayload>`